examples/Popover.focus.tsx

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Button, Flex, Popover, Surface, Text } from '@itwin/itwinui-react';
+
+export default () => {
+  const idPrefix = React.useId();
+  const headingId = `${idPrefix}-label`;
+  const descriptionId = `${idPrefix}-description`;
+
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    <Popover
+      applyBackground
+      aria-labelledby={headingId}
+      aria-describedby={descriptionId}
+      visible={isOpen}
+      onVisibleChange={setIsOpen}
+      style={{ maxWidth: '45ch', border: 'none' }}
+      content={
+        <Surface elevation={0}>
+          <Surface.Header>
+            <Text as='h3' id={headingId} variant='leading'>
+              Terms and conditions
+            </Text>
+          </Surface.Header>
+          <Surface.Body isPadded>
+            <Flex flexDirection='column' alignItems='start'>
+              <Text as='p' id={descriptionId}>
+                (TODO: improve text) Lorem ipsum dolor sit amet consectetur
```

Comment:
@gretanausedaite @rohan-kadkol @siddhantrawal Can somebody suggest better text or a different example? I dunno how this has 3 approvals with TODO text 😅

Reply:
> Can somebody suggest better text or a different example?

How about a "see image preview" popover? If we click on the trigger button that says "show image preview", the popover shows the image preview along with an "OK" button and a "Download" button.

> I dunno how this has 3 approvals with TODO text 😅

I thought that TODO was for you so you will handle it yourself 😅 So I gave a preemptive review to not block the PR because everything else LGTM.

Reply:
> Can somebody suggest better text or a different example?

Some sliders to "tune settings", eg. quality, grain, color, saturation, etc. (like in photoshop) with ok or save / cancel buttons at the bottom.

Reply:
> I dunno how this has 3 approvals with TODO text

We have more than one todo in repo that is supposed to be solved later.

Reply:
updated the example to show a settings dialog

![](https://github.com/iTwin/iTwinUI/assets/9084735/10ea6505-9635-4996-b6ca-8cd11ff7146b)

Reply:
It's good. We have this in css fieldset tests if you want to switch inputs but you don't have to. This example is to show focus control, it does not have to be useful or realistic.
![image](https://github.com/iTwin/iTwinUI/assets/83585998/bb8bb898-49cb-4c74-b9fe-d77924060d3b)